packages/core/src/browser/status-bar/status-bar.tsx

```diff
@@ -113,16 +141,17 @@ export class StatusBarImpl extends VirtualWidget implements StatusBar {
                 rightEntries.push(this.renderElement(entry));
             }
         });
-        const leftElements = h.div({ className: 'area left' }, VirtualRenderer.flatten(leftEntries));
-        const rightElements = h.div({ className: 'area right' }, VirtualRenderer.flatten(rightEntries));
-        return VirtualRenderer.flatten([leftElements, rightElements]);
+        const leftElements = <div className="area left"><React.Fragment>{leftEntries}</React.Fragment></div>;
```

Comment:
`React.Fragment` is redundant here, `leftEntries` already wrapped into `div`, the same for `rightEntries`

Reply:
Without wrapping React.Fragment I get the usual warning:
```
Warning: Each child in an array or iterator should have a unique "key" prop.

Check the top-level render call using <div>. See https://fb.me/react-warning-keys for more information.
    in div
```

Reply:
Btw. `React.Fragment` does not replace `div`. Actually it does not render a html tag at all. As far as I understood one can use it to group a list of elements without setting key attribute.

Reply:
i am not sure `React.Fragment` is proper fix. `https://fb.me/react-warning-keys` does not tell anything about it. They say there that each child of `leftEntries` and `rightEntries` should have a key. Are you sure that `React.Fragment` does the same?

Reply:
As I said, as far as I understood the following blog post, yes:
https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

Reply:
cool, thanks for checking!